query/sources/common/src/main/java/org/codice/ddf/admin/sources/utils/SourceValidationUtils.java

```diff
@@ -36,14 +36,15 @@ public SourceValidationUtils(ConfiguratorSuite configuratorSuite) {
     serviceCommons = new ServiceCommons(configuratorSuite);
   }
 
+  @SuppressWarnings("squid:S1135" /* Remove when TODO is completed */)
   private boolean findSourceNameMatch(String servicePid, String sourceName) {
     Source source =
         sourceUtilCommons
             .getAllSourceReferences()
             .stream()
             .map(ConfiguredService.class::cast)
             // TODO: 8/22/17 phuffer - this doesn't work for OpenSearch - opensearch sources get
```

Comment:
I will verify whether or not this is still a problem. I was able to remove another open search todo recently that was fixed magically.

Reply:
This TODO can be removed. Verified OS sources return the correct pids here.